boards/arm/arduino_zero/doc/arduino_zero.rst

```diff
@@ -0,0 +1,126 @@
+.. _arduino_zero:
+
+Arduino/Genduino Zero
+################
+
+Overview
+********
+
+The Arduino Zero is a maker-friendly development board with an
+integrated debugger.
```

Comment:
maybe elaborate this a bit with:

    The Arduino Zero is a maker-friendly development board with
    Atmel’s Embedded Debugger (EDBG), which provides a full
    debug interface without the need for additional hardware.

Reply:
Done